### DISCUSSION TITLE
PLEASE READ THISSSS

### DISCUSSION THREAD

**Discussion by bradenM03** [EtherealEmil/UsualScrap#44] [Ideas]
I'm literally begging you bro. Please add additional configurations to the shift controller. The concept is actually genius, but the range is just too short to make it worth using. Me and my friends have been using this mod religiously for a while now and most of the items are perfect, but the shift controller is just too weak. Just please allow us to configure the range before it glitches. Also if you could add an option for the coordinate that you set with the controller to remain saved when you drop the controller. There are so many fun uses with it that are ruined by the fact that you have to keep it in your inventory. For example, on rend you can save your location at the top of the mountain near fire exit, drop down to the bottom and leave the controller at the door, then when you need to run loot you can pick it up and teleport up the mountain. (This might seem OP but the jetpack does the same thing) This feature would also let you leave the controller on the ground in the facility once you're near the max range, that way you can explore a bit deeper and come back to it to teleport back to the entrance. Anyways, that's about it, just wanted to say thanks for all the work you've put into this mod, its one of my favorites and my friends and I have been loving it. Also a little sidenote, the scrap dropped from using the toolbox doesn't have any value, not sure if this was fixed in the last couple updates but I figured id mention it. Still a really cool item tho, and has helped with a few difficult turret spawns.

**Comment by EtherealEmil** [comment]
First I really appreciate y'all and I'm glad y'all have been enjoying the mod. Second, configs may or may not be added I'd have to tweak the code a bit to be a bit more interchangeable in terms of changing the values but it's definitely something I will consider. Third, you should be able to drop it and the location stay saved unless it runs out of battery, I've noticed a bug where if it is dropped INSIDE the ship the coordinates get wiped for some reason and am trying to figure that out but it should be fine to drop it everywhere else without it wiping, beside the ship issue if it isn't keeping the coordinates when dropped everywhere else either let me know or I'll check next time I do some play testing. Lastly, the scrap appearing to have no value is an issue I've been trying to fix but just know it only appears that way for clients and is only visual, when sold you will receive the credits.

**Reply by bradenM03** [reply]
Ah okay that makes sense, I haven't used the controller much, but I was testing it mostly in the ship so that explains why the coordinates were being wiped. As for the toolbox scrap we use the sell from terminal mod so I didn't even realize it actually had value. Thats good to know. I'll def be using the shift controller a lot more now, hopefully you can still add that config option to it. Thanks again.

**Reply by bradenM03** [reply]
Also configurations to the battery life of the controller would be nice if possible, but I know im asking too much now lol.
